sweepai/handlers/on_ticket.py

```diff
@@ -357,13 +357,22 @@ def fetch_file_contents_with_retry():
             )
 
             # COMMENT ON ISSUE
-            # TODO: removed issue commenting here
             logger.info("Fetching files to modify/create...")
             file_change_requests, create_thoughts, modify_thoughts = sweep_bot.get_files_to_change()
 
-            sweep_bot.summarize_snippets(create_thoughts, modify_thoughts)
-
-            file_change_requests = sweep_bot.validate_file_change_requests(file_change_requests)
+            # Add a new FileCreation object for the issue_template file
+            issue_template_content = (
```

Comment:
Move 



            # Add a new FileCreation object for the issue_template file
            issue_template_content = (
                "# Issue Title\n\n"
                "Please provide a clear and concise title for the issue.\n\n"
                " # Issue Description\n\n"
                "Please provide a detailed description of the issue. Include any context or information that could be relevant to resolving the issue.\n\n"
                "# Steps to Reproduce (if applicable)\n\n"
                "If this issue is a bug, please provide detailed steps for reproducing the issue.\n\n"
                "1.\n2.\n3.\n\n"
                "# Expected Behavior\n\n"
                "Please describe what you expect to happen or see as a result of resolving this issue.\n"
            )

this to create_config_pr